scripts/lib/cluster.sh

```diff
@@ -96,6 +101,10 @@ function up-kops-cluster {
         let RETRY_ATTEMPT=RETRY_ATTEMPT+1
         echo "In attempt# $RETRY_ATTEMPT, waiting for cluster validation"
     done
+
+    kubectl delete daemonset ebs-csi-node -n kube-system || true
+    kubectl delete deployment coredns-autoscaler -n kube-system || true
```

Comment:
nit: no need to "or" with true. If you want to log if this fails, then you should "or" with a log statement

Reply:
I'll remove the "or" statements since they are not needed now